libs/application/template-api-modules/src/lib/modules/shared/shared.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable, Inject } from '@nestjs/common'
 import { ConfigService } from '@nestjs/config'
 
+import { ChargeResult } from '@island.is/api/domains/payment'
```

Comment:
Should this type be defined closer to the PaymentService which returns it? Not recommended to import libs from other apps (application-system vs api libs)

Reply:
I moved the PaymentService from libs to client from earlier comment. I suppose we are good on this one.